packages/service/db/migrations/20210720160437_add_document_updates_queue.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+CREATE TABLE document_updates_queue (
+  id uuid DEFAULT uuid_generate_v4() NOT NULL PRIMARY KEY,
+  document_id uuid NOT NULL,
```

Comment:
Would it be better to name the table updates? I do not think it is necessary to specify the relationship with the document table in the table name as it is tied with the primary key. I also wouldn't add `query` in the name either.